`

Ishwarpatra opened the pull request:
#14 Refactor: Made the code smarter and safer to use.

I've swapped out basic data types like dictionaries for more descriptive "smart" objects (`Artist`, `Recommendation`, etc.). This makes the code easier to understand, prevents common errors, and will make it much simpler to add new features later on.

Commit:
`
Refactor: Use DDD for Type Safety & Clarity

Replaced primitive types with domain objects (Artist, ArtistWeight, Recommendation) to improve type safety and make the code more explicit and maintainable.
`

Ishwarpatra commented:
please check this ,add accept it

dettbarn commented:
@Ishwarpatra did you forget to push something? The PR only contains one added empty file, everything else unchanged

Ishwarpatra commented:
Okay I will give the request again with change

Commit:
`
Refactor: Use DDD for Type Safety & Clarity

Replaced primitive types with domain objects (Artist, ArtistWeight, Recommendation) to improve type safety and make the code more explicit and maintainable.
`

Ishwarpatra commented:
@dettbarn  now check it
